src/common/Pagination/styled.js

```diff
@@ -13,8 +13,27 @@ export const StyledPagination = styled.section`
 
   @media (max-width: ${bpMobile}px) {
     gap: 8px;
-    margin: 0 0 31px;
   }
+
+  ${({ location }) =>
+    location === "PopularMovies" &&
```

Comment:
It's a small thing, but we should use lowcase here: `popularPeople`. Just a convention rule